.buildkite/scripts/steps/integration_tests.sh

```diff
@@ -8,22 +8,13 @@ MAGE_TARGET="${2:-"integration:test"}"
 MAGE_SUBTARGET="${3:-""}"
 
 
-# Override the agent package version using a string with format <major>.<minor>.<patch>
-# There is a time when the snapshot is not built yet, so we cannot use the latest version automatically
-# This file is managed by an automation (mage integration:UpdateAgentPackageVersion) that check if the snapshot is ready.
-OVERRIDE_AGENT_PACKAGE_VERSION="$(cat .package-version)"
```

Comment:
I don't think we should stop doing that!
We need https://github.com/elastic/elastic-agent/pull/5100 to be merged instead.